In-house patch to remove unnecessary dependencies from Glance's
requirements files. The specific reasons are as follows:

lxml		No longer applicable
		(upstream commit b7f9120cd4623d8284c4edfea760ef10e1193492)

passlib		No longer applicable
		(upstream commit b31874284212d49560ddf4f36e5c9ca5d221f605)

pycrypto	Not applicable to Solaris (M2Crypto used instead)

wsgiref		No longer applicable
		(upstream commit b31874284212d49560ddf4f36e5c9ca5d221f605)

--- glance-2013.2.3/glance.egg-info/requires.txt.orig	2014-04-03 11:46:54.000000000 -0700
+++ glance-2013.2.3/glance.egg-info/requires.txt	2014-05-24 23:04:42.210143004 -0700
@@ -6,20 +6,16 @@
 PasteDeploy>=1.5.0
 Routes>=1.12.3
 WebOb>=1.2.3,<1.3
-wsgiref>=0.1.2
 boto>=2.4.0,!=2.13.0
 sqlalchemy-migrate>=0.7.2
 httplib2
 kombu>=2.4.8
-pycrypto>=2.6
 iso8601>=0.1.8
 oslo.config>=1.2.0
 python-swiftclient>=1.5,<2.0.0
-lxml>=2.3
 Paste
-passlib
 jsonschema>=1.3.0,!=1.4.0
 python-cinderclient>=1.0.6
 python-keystoneclient>=0.3.2
 pyOpenSSL>=0.11
-six>=1.4.1
\ No newline at end of file
+six>=1.4.1
--- glance-2013.2.3/requirements.txt.orig	2014-04-03 11:43:55.000000000 -0700
+++ glance-2013.2.3/requirements.txt	2014-05-24 23:05:14.053235541 -0700
@@ -12,29 +12,20 @@
 PasteDeploy>=1.5.0
 Routes>=1.12.3
 WebOb>=1.2.3,<1.3
-wsgiref>=0.1.2
 argparse
 boto>=2.4.0,!=2.13.0
 sqlalchemy-migrate>=0.7.2
 httplib2
 kombu>=2.4.8
-pycrypto>=2.6
 iso8601>=0.1.8
 oslo.config>=1.2.0
 
 # For Swift storage backend.
 python-swiftclient>=1.5,<2.0.0
 
-# Note you will need gcc buildtools installed and must
-# have installed libxml headers for lxml to be successfully
-# installed using pip, therefore you will need to install the
-# libxml2-dev and libxslt-dev Ubuntu packages.
-lxml>=2.3
-
 # For paste.util.template used in keystone.common.template
 Paste
 
-passlib
 jsonschema>=1.3.0,!=1.4.0
 python-cinderclient>=1.0.6
 python-keystoneclient>=0.3.2
